stubs/inertia-react/resources/js/Components/Dropdown.jsx

```diff
@@ -70,13 +70,11 @@ const Content = ({ align = 'right', width = '48', contentClasses = 'py-1 bg-whit
     );
 };
 
-const DropdownLink = ({ href, method, as, children }) => {
+const DropdownLink = ({ className, children, ...props }) => {
     return (
         <Link
-            href={href}
-            method={method}
-            as={as}
-            className="block w-full px-4 py-2 text-left text-sm leading-5 text-gray-700 dark:text-gray-300 hover:bg-gray-100 dark:hover:bg-gray-800 focus:outline-none focus:bg-gray-100 dark:focus:bg-gray-800 transition duration-150 ease-in-out"
+            {...props}
+            className={`block w-full px-4 py-2 text-left text-sm leading-5 text-gray-700 dark:text-gray-300 hover:bg-gray-100 dark:hover:bg-gray-800 focus:outline-none focus:bg-gray-100 dark:focus:bg-gray-800 transition duration-150 ease-in-out ${className}`}
```

Comment:
`className` can be undefined and will print that in this expression, so might want to handle that case.